README.md

```diff
@@ -8,6 +8,8 @@
 - Add a book.
 - Remove a selected book.
 
+![screenshot](./Captura.PNG)
+
 
 ## Built With
```

Comment:
**[OPTIONAL]** You have well crafted your README.md. It really looks nice and clean!:champagne:

You can also consider those options to make more professional (take a look at [Microverse standard template](https://github.com/microverseinc/readme-template)):
- [ ] separate Major languages, Frameworks, Technologies.

For instance:
- Major languages: JavaScript, HTML, CSS
- Frameworks: React...

- [ ] add live demo link (you can easily upload it to netlify). If you need some tutorial, please check out this amazing 22 min video from Brad Traversy on [YouTube](https://www.youtube.com/watch?v=bjVUqvcCnxM&t=334s). Watching only the first half would be more than enough:wink: